WidgetExtension/Up Next/UpNextStyles.swift

```diff
@@ -4,6 +4,128 @@ import SwiftUI
 let darkBackgroundColor = Color(UIColor.tertiarySystemFill)
 let lightBackgroundColor = Color(.clear)
 
+struct PCWidgetColorScheme {
+    let topBackgroundColor: Color
+    let bottomBackgroundColor: Color
+    let topButtonBackgroundColor: Color
+    let bottomButtonBackgroundColor: Color
+    let topTextColor: Color
+    let bottomTextColor: Color
+    let topButtonTextColor: Color // foreground color?
```

Comment:
Can the comments or question marks be removed here? I guess I'm not sure whether these are notes or actually describe the colors in some way.

Reply:
I think that was a question for myself... if I should rename `TextColor` to `ForegroundColor` 😅 I'll remove the comment.